app/main.py

```diff
@@ -1,3 +1,2 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    return int(phrase.lower().count(letter.lower()))
```

Comment:
The return statement should not cast the result of `count` to an integer. The `count` method already returns an integer, so the `int()` conversion is unnecessary and misleading. Simply return `phrase.lower().count(letter.lower())`.